pyvista/core/utilities/features.py

```diff
@@ -32,6 +62,10 @@ def voxelize(mesh, density=None, check_surface=True):
         algorithm first checks to see if the surface is closed and
         manifold. If the surface is not closed and manifold, a runtime
         error is raised.
+        
+    enclosed : bool, default: False
```

Comment:
Can we add an example to the docstring with `enclosed=True`? It would be nice to have two figures one after the other to easily see / compare the difference. Something similar to what's shown in the this PR.